src/pybind/rados/setup.py

```diff
@@ -122,6 +122,22 @@ def check_sanity():
     def cythonize(x, **kwargs):
         return x
 
+if '--without-cython' in sys.argv:
```

Comment:
under what condition, user would use `setup.py` without cython? and we don't have `rados.c` in our repo yet, right?